src/App.js

```diff
@@ -31,10 +33,36 @@ class App extends React.Component {
     })
   }
 
-  render(){
+  // handleNumberHorns = event => {
+  //   let newNumsArr = totalImageArray;
+  //   let selected = event.target.horns;
+
+  //   this.setState({
+  //     selected: event.target.horns
+  //   })
+  // }
+
+  render() {
+    console.log(this.state);
     return (
       <>
         <Header />
+
+        <Container>
+          <Form>
+            <Form.Group controlId="selected">
+              <Form.Label>Beast with How many Horns would you like to see!</Form.Label>
+              <Form.Select onChange={this.handleNumberHorns}>
+                <option>Make a Choice</option>
+                <option value="one">One</option>
+                <option value="two">Two</option>
+                <option value="three">Three</option>
+                <option value="hundred">Hundred</option>
+              </Form.Select>
+            </Form.Group>
+          </Form>
+        </Container>
```

Comment:
- You are on the right track here, but we most likely will want this Form logic elsewhere, perhaps Main will also work nicely here 👍

Reply:
thank you for the feedback